schematics/src/utils/registration.ts

```diff
@@ -180,6 +185,17 @@ export function addImportToFile(options: { module?: string; artifactName?: strin
   };
 }
 
+export function addLazyExportToBarrelFile(options: { path?: string; name?: string; artifactName?: string }): Rule {
+  const barrelFile = `${options.path}/index.ts`;
+  return host => {
+    if (!tsquery(readIntoSourceFile(host, barrelFile), `Identifier[name=${options.artifactName}]`).length) {
+      const exportRecorder = host.beginUpdate(barrelFile);
+      insertExport(exportRecorder, options.artifactName, `./${options.name}/${options.name}.component`);
```

Comment:
for this rule the option `moduleImportPath` seems to be set correctly. Could you use it like the `insertImport` in the previous helper `addImportToFile` instead of combining the path directly?